crates/orderbook/src/database/orders.rs

```diff
@@ -356,6 +359,70 @@ impl OrderStoring for Postgres {
         .try_collect()
         .await
     }
+
+    async fn order_status(&self, order_uid: &OrderUid) -> Result<Status> {
+        let mut ex = self.pool.begin().await.context("could not init tx")?;
+        let timer = super::Metrics::get()
+            .database_queries
+            .with_label_values(&["latest_order_event"])
+            .start_timer();
+        let status = database::order_events::get_latest(&mut ex, &ByteArray(order_uid.0))
+            .await
+            .context("could not fetch status")?;
+        timer.stop_and_record();
+
+        let fetch_solvers = || async move {
+            let timer = super::Metrics::get()
+                .database_queries
+                .with_label_values(&["load_latest_solver_competition"])
+                .start_timer();
+            let competition = database::solver_competition::load_latest_competition(&mut ex)
+                .await
+                .context("could not fetch latest competition")?
+                .context("competition not found for the order")?
+                .json;
+            timer.stop_and_record();
+
+            let competition: SolverCompetitionDB = serde_json::from_value(competition)
+                .context("could not parse solver competition data")?;
+
+            let solvers = competition
+                .solutions
+                .into_iter()
+                .filter_map(|solution| {
+                    solution
+                        .orders
+                        .iter()
+                        .any(|o| match o {
+                            model::solver_competition::Order::Colocated { id, .. }
+                            | model::solver_competition::Order::Legacy { id, .. } => {
```

Comment:
Is it possible to include in the response whether the order is included in a given solution?
I ask because I had the understanding that the `solved` state would include the solvers listed by ranking.
Given that this state denotes when your order ISN'T in the winning solution, it could have been in one or more of the runner ups.

Based on this, there was a UI proposal to show which solutions included the user order:

<img width="970" alt="image" src="https://github.com/user-attachments/assets/e93ed2d2-f33c-4b7d-a806-f21d2db43ca4">

Although it's not used in the most up to date proposal:

<img width="572" alt="image" src="https://github.com/user-attachments/assets/6b0d8fb6-df1b-4bad-acd6-2f74a1537f82">

So, not a must, just a good to have if possible.

Reply:
> Is it possible to include in the response whether the order is included in a given solution?

The retrieved competition contains all the solutions received(sorted by ranking), but only those containing the specified order uid are filtered in. So if the solvers list is empty, the latest competition doesn't contain that order. Do you expect to receive the solvers list with a flag on whether the order was included in a specific solution?

Reply:
It would be great to have all the solvers always, even if the order was not included.
This way we can have more transparency in the competition and users can see when a solver included their order or not.
The flag as proposed is enough 👍